console-extensions.json

```diff
@@ -59,23 +59,14 @@
       "component": { "$codeRef": "AddressDetailsContainer.App" }
     }
   },
-  {
-    "type": "console.navigation/section",
-    "properties": {
-      "id": "activemq",
-      "perspective": "admin",
-      "name": "ActiveMQ Artemis",
-      "dataAttributes": { "data-quickstart-id": "qs-nav-home" }
-    }
-  },
   {
     "type": "console.navigation/href",
     "properties": {
       "id": "brokers",
-      "name": "ActiveMQ Artemis Self Provisioning",
+      "name": "Brokers",
       "href": "/k8s/all-namespaces/brokers",
       "perspective": "admin",
-      "section": "activemq"
+      "section": "workloads"
     }
   }
-]
+]
```

Comment:
This removes the top-level plugin section. Maybe we can keep it along with `workloads` so people can quickly find the spp plugin? (maybe adding an `About` item)